deployment/node-feature-discovery/values.yaml

```diff
@@ -180,7 +181,19 @@ worker:
 
   nodeSelector: {}
 
-  tolerations: {}
+  tolerations:
+    - operator: "Exists"
+      effect: "NoSchedule"
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: node-role.kubernetes.io/master
+                  operator: DoesNotExist
+            - matchExpressions:
+                - key: node-role.kubernetes.io/worker
+                  operator: Exists
```

Comment:
See above comment about tolerations. I suggest to make this change in a separate PR, changing the templates at the same time.

Reply:
I can take that as a TODO for sure, this add NFD support for people using it on single node deployments and edge cases, so is on my priority list

Reply:
K, thanks